Test.C

```diff
@@ -235,6 +235,57 @@ int main(){
         // Selling equipment
         // decrease amount in index amount if things are sold
         // increase credit if things are sold
+        // So this is a lot of pseudo code or not finished but idk how to do it fully but it a good framework 
+        // assumes that you can sell the equipment for the same price that is bought 
+        int equipSellSelect;
+        int equipSellAmount;
+        int equipSellTransaction;
+        int equipSellAnswer;
+        
+        printf("|----------------------------- Equipment Selling ------------------------------|\n");
+        printf("Equipment that you own:\n");
+        for(int i = 0; i < NUM_EQUIPMENT; i++){
```

Comment:
This does not follow the style guide, check [here](https://github.com/Phabbits/CNIT315/wiki/how-to-contribute#spaces-horizontal-spacing).
```suggestion
        for(int i=0; i<NUM_EQUIPMENT; i++){
```